node/src/components/fetcher.rs

```diff
@@ -603,6 +603,42 @@ impl ItemFetcher<BlockHeader> for Fetcher<BlockHeader> {
     }
 }
 
+impl ItemFetcher<BlockHeadersBatch> for Fetcher<BlockHeadersBatch> {
+    const SAFE_TO_RESPOND_TO_ALL: bool = false;
```

Comment:
I don't think it matters, but I wonder if this is actually `true` :).

Reply:
Done in [`9c7c3e2` (#3020)](https://github.com/casper-network/casper-node/pull/3020/commits/9c7c3e2c3492f1f6f3d6583055ec6d6c33cfe53b)